docs/04-api-reference/front-commerce-core/testing.mdx

```diff
@@ -0,0 +1,58 @@
+---
+title: "Testing"
+sidebar_position: 9
+description:
+  "Testing is an important part of the development process. It helps to ensure that
+the code is working as expected and that it is not breaking any existing
+functionality. In this section, we will cover all the tools and utilities that
+we provide for testing."
+---
+
+<p>{frontMatter.description}</p>
+
+<SinceVersion tag="3.4" />
+
+## [`createFrontCommerceProviderStub`](https://gitlab.blackswift.cloud/front-commerce/front-commerce/-/blob/main/packages/core/react/FrontCommerceProviderStub.tsx?ref_type=heads)
```

Comment:
**question:** Should the link point to a specific commit instead of `main`? Or do we assume we'll update the documentation if this come to change anyway?